### ISSUE TITLE
Unsupported conditions are present - Safe to ignore these warnings ? 

### ISSUE THREAD

**Issue by shyamjos** [intel/thermal_daemon#282]
Hi I am getting these warnings on Asus x412ua running fedora 33

```
● thermald.service - Thermal Daemon Service
     Loaded: loaded (/usr/lib/systemd/system/thermald.service; enabled; vendor preset: enabled)
     Active: active (running) since Wed 2020-11-18 21:46:55 EST; 10h left
   Main PID: 1054 (thermald)
      Tasks: 4 (limit: 14150)
     Memory: 4.5M
        CPU: 150ms
     CGroup: /system.slice/thermald.service
             └─1054 /usr/sbin/thermald --systemd --dbus-enable --adaptive

Nov 18 21:46:57 localhost-live thermald[1054]: Unsupported conditions are present
Nov 18 21:46:57 localhost-live thermald[1054]: Unable to find a sensor for \_SB_.PCI0.LPCB.EC0_.SEN1
Nov 18 21:46:57 localhost-live thermald[1054]: Unable to find a sensor for \_SB_.PCI0.LPCB.EC0_.SEN1
Nov 18 21:46:57 localhost-live thermald[1054]: Unable to find a sensor for \_SB_.PCI0.LPCB.EC0_.SEN1
Nov 18 21:46:57 localhost-live thermald[1054]: Unable to find a sensor for \_SB_.PCI0.LPCB.EC0_.SEN1
Nov 18 21:46:57 localhost-live thermald[1054]: 22 CPUID levels; family:model:stepping 0x6:8e:9 (6:142:9)
Nov 18 21:46:57 localhost-live thermald[1054]: Polling mode is enabled: 4
Nov 18 21:46:57 localhost-live thermald[1054]: sensor id 9 : No temp sysfs for reading raw temp
Nov 18 21:46:57 localhost-live thermald[1054]: sensor id 9 : No temp sysfs for reading raw temp
Nov 18 21:46:57 localhost-live thermald[1054]: sensor id 9 : No temp sysfs for reading raw temp
``` 
Model name: Intel(R) Core(TM) i3-7020U CPU @ 2.30GHz

Is this warnings safe to ignore ?

**Comment by spandruvada** [comment]
Can't say.
#systemctl stop thermald
#thermald --no-daemon --loglevel=info --adaptive

In another window
grep -r . /sys/class/thermal/thermal_zone*

also attach output of acpidump > acpi.out

**Comment by shyamjos** [comment]
[acpi.out.txt](https://github.com/intel/thermal_daemon/files/5561985/acpi.out.txt)
[grep_thermal_zone.txt](https://github.com/intel/thermal_daemon/files/5561987/grep_thermal_zone.txt)
[thermald_debug.log](https://github.com/intel/thermal_daemon/files/5561988/thermald_debug.log)

BTW I have been facing throttling issue due to `ASUS PL2 problem` , so right now I am using patched version of thermald provided by distro maintainer (fedora), which is probably a fix from here #280

more info here : https://bugzilla.redhat.com/show_bug.cgi?id=1892534

**Comment by spandruvada** [comment]
They are safe to ignore.
